checkbox-provider-ce-oem/tox.ini

```diff
@@ -6,18 +6,27 @@ skipsdist=True
 [testenv]
 allowlist_externals = rm
 commands =
-    pip -q install -e ../../checkbox-ng
+    pip -q install ../../checkbox-ng
     # Required because this provider depends on checkbox-support parsers & scripts
-    pip -q install -e ../../checkbox-support
+    pip -q install ../../checkbox-support
     rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-resource.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-base.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-certification-client.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-certification-server.provider
+    rm -f /var/tmp/checkbox-providers-develop/plainbox-provider-docker.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-gpgpu.provider
+    rm -f /var/tmp/checkbox-providers-develop/com.canonical.certification.checkbox-provider-iiotg.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-sru.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-tpm2.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-ce-oem.provider
     # Install all providers in develop mode to make sure everything works fine
```

Comment:
@hanhsuan 
following checkbox providers are not needed for validate the ce-oem-provider, could you please remove them?
- plainbox-provider-docker
- checkbox-provider-gpgpu
- checkbox-provider-iiotg
- checkbox-provider-sru
- checkbox-provider-tpm